public/app/views/shared/_digital.html.erb

```diff
@@ -1,11 +1,19 @@
-<% if @result['json']['representative_file_version'].present? %>
+<% if record['json']['representative_file_version'].present? %>
 <div class="available-digital-objects">
   <div class="objectimage">
     <div class="panel panel-default">
       <%= render partial: 'shared/representative_file_version', locals: {
-        :uri => @result['json']['representative_file_version']['file_uri'],
```

Comment:
@mark-cooper I was wondering if I was going against a convention by calling the instance variable here and elsewhere. What is the idea, only use instance variables in the top-level views named after the controller actions and only locals in child templates?